bpf/init.sh

```diff
@@ -301,6 +304,41 @@ function bpf_load()
 	return $RETCODE
 }
 
+function bpf_load_cgroups()
+{
+	OPTS=$1
+	IN=$2
+	OUT=$3
+	PROG_TYPE=$4
+	WHERE=$5
+	SEC=$6
+	CALLS_MAP=$7
+	CGRP=$8
+	BPFMNT=$9
+
+	OPTS="${OPTS} ${OPTS_DIR} -DCALLS_MAP=${CALLS_MAP}"
+	bpf_compile $IN $OUT obj "$OPTS"
+
+	TMP_FILE="$BPFMNT/tc/globals/cilium_cgroups_$WHERE"
+	rm -f $TMP_FILE
+
+	cilium-map-migrate -s $OUT
+	set +e
+	tc exec bpf pin $TMP_FILE obj $OUT type $PROG_TYPE attach_type $WHERE sec $SEC
+	RETCODE=$?
+	set -e
+	cilium-map-migrate -e $OUT -r $RETCODE
+
+	if [ "$RETCODE" -eq "0" ]; then
+		set +e
+		bpftool cgroup attach $CGRP $WHERE pinned $TMP_FILE
```

Comment:
We should probably mention about the `bpftool` dependency at https://docs.cilium.io/en/latest/install/system_requirements/.

Reply:
Makes sense, we already have this dependency for sockmap today (bpftool is part of our container image). I'll add a todo item.